### ISSUE TITLE
Can we increase the length of the answers ?

### ISSUE THREAD

**Issue by ankitkr3** [patil-suraj/question_generation#8]
Hi patil,

Thanks a lot for this helpful git, how can we increase the length of the answer from words to a sentence or few sentences ?

**Comment by patil-suraj** [comment]
Hi @ankitkr3 , the answer extraction is also trained using SQuAD data and it doesn't have long answers, so it's kind of a dataset limitation. I'm also thinking about how to increase the length of answers, I'll update this issue once I find a proper solution

**Comment by ankitkr3** [comment]
okay, will be waiting,  there's a facebook data for long answers i think.

**Comment by patil-suraj** [comment]
Yes, that's ELI5 dataset but it's abstracive QA data.

**Comment by ankitkr3** [comment]
any update on long answer generation?

**Comment by patil-suraj** [comment]
Hi @ankitkr3 , this is going to take some time, busy with other tasks now, if you are interested then take a look at Natural Questions dataset. It has both long and short answers. The dataset is available in `nlp` library. Note that it's a really big dataset, so processing will take time. Feel free to give it a try.

**Comment by ankitkr3** [comment]
okay @patil-suraj will try and update you. thanks.

**Comment by darshanapdas** [comment]
Hi,

I'm also attempting to train this model for the natural questions dataset. How do you think we need to train the answer extraction model for longer sentences. Do we consider the long-answer in the NQ dataset as both context and answer? If that is the case how do we then input sentences to the model? as in should we give the whole paragraph or combinations of sentences?

TLDR: how should one approach the training for the answer extraction model for NQ dataset?

Thanks,
Darshana
